src/tracing/instrumentation/httpServer.js

```diff
@@ -65,10 +64,12 @@ function requestListener(req, res) {
   }
 
   res.on('finish', function() {
+    var urlParts = req.url.split('?');
     span.data = {
       http: {
         method: req.method,
-        url: discardUrlParameters(req.url),
+        url: urlParts.shift(),
```

Comment:
Could we continue to use `discardUrlParameters`? It also discarded matrix parameters and the hash sequence for safety.